\doublespacing

\section{Introduction}

The natural progression after biomarker validation was to identify the role of \acrfull{ifit1} within the body, and to ascertain whether \acrshort{ifit1} had an involvement in any current mechanistic models of hypertension.

The Ifit family of proteins includes four canonical members within humans (Ifit1, Ifit2, Ifit3, and Ifit5), and is widely conserved in mammals, amphibians, and fish \cite{Pichlmair2011,Diamond2013}. Within rats specifically there are 4 ifit proteins; Ifit1, Ifit1b, Ifit2, and Ifit3 \cite{Zhou2013}. Interestingly, Ifit proteins cannot be found in lower order animals, such as the fruit fly (\textit{Drosophila melanogaster}), nematode (\textit{Caenorhabditis elegans}), yeast (\textit{Saccharomyces cerevisiae}) or plants \cite{Chebath1983,Kusari1987} (Figure \ref{fig:ifitfamily}). 

\begin{figure*}[!htbp]
\centering
\includegraphics[width=1\textwidth]{chapter5/ifit_tree.jpg}
\caption[Phylogeny of Ifit family proteins]{Phylogeny of Ifit family proteins, as taken from Zhou \textit{et al} 2013, and generated by MEGA4.0 \cite{Zhou2013,Tamura2007}}
\label{fig:ifitfamily}
\end{figure*}

Generally Ifit family genes are less abundantly expressed whilst in the absence of stimuli. However their expression rapidly increases during stimulation with IFN, viral infection, or other \acrfull{pamp} recognition. Most of the Ifit family of genes have two exons and contain between two or three \acrfull{isre} as part of their promoter regions \cite{Fensterl2011}. The presence of ISREs within the promoter of Ifits serves to explain their low basal expression levels and rapid IFN-dependent induction \cite{Sarkar2004}. These \acrshort{isre} are crucial cis-acting response elements that are recognised by IFN-Stimulated Gene Factor 3 (ISGF3), a factor that is induced by IFNs and various other stimuli \cite{Gobin1997}. The main inducers of the ifit genes therefore centre around the interferons (IFNs) and include; type I IFNs (IFN-$\alpha$/$\beta$), type III IFNs (IFN-$\lambda$), and to a much weaker degree Type II IFNs (IFN-$\gamma$) \cite{Der1998,Kohli2012}. 

The Ifit protein family are mostly described for their role in antiviral immunity. While none of the ifit family of proteins have any known enzymatic activity, they can inhibit replication of viruses by binding and modulating the functions of cellular and viral proteins and RNAs. All Ifits commonly share a tetratricopeptide (TPR) domain containing a 34 amino acid motif with the consensus sequence $[WLF]-X_{(2)}-[LIM]-[GAS]-X_{(2)}-[YLF]-X_{(8)}-[ASE]-X_{(3)}-[FYL]-X_{(2)}-[ASL]-X_{(4)}-[PKE]$. This motif folds into a resulting helix-turn-helix structure giving rise to a series of unique concave and convex surfaces that permit binding to a diverse range of ligands. TPR domains themselves are conserved in all kingdoms of life, and are considered to function as protein and peptide recognition sites. The discovery of Ifit's ability to bind RNA has served to broaden the known ligand spectrum of TPR motifs to include nucleic acids.  

Through these TPR domains, ifit family proteins can modulate protein interactions \cite{DAndrea2003}. For example, both Ifit1 and Ifit2 are involved in a nonspecific antiviral effect via their direct interaction with Eukaryotic Initiation Factor 3 (eIF3) translation complex. The eIF3 complex is essential for translation initiation for several reasons; recruitment of mRNA, scanning of the mRNA for a start codon, and delivery of tRNA to the translation machinery \cite{Merrick2004}. Through binding with the subunits of the eIF3 protein, Ifits can block mRNA recruitment and tRNA delivery \cite{Hui2003}. This leads to around 60\% suppression of translation within the cell during protein synthesis which can indeed be detrimental to the host cell, however this also prevents host-dependent viral replication \cite{Hui2005,Terenzi2006,Guo2000}. 

As well as inhibiting endogenous aspects of the viral replication pathway, recent evidence has shown the ability of Ifit family proteins to directly bind viral \acrshort{rna}. Generally speaking, host cytoplasmic RNAs are single stranded and mRNAs and rRNAs/tRNAs contain either a N-7-methylated guanosine or 5'-monophosphate cap, bound by a 5'-to-5' triphosphate bridge to the first base respectively. Certain higher order eukaryotes see a further modification to the mRNA in the form of a methylation of the first ribose \cite{Topisirovic2011,Gebauer2004}. These modifications confer more than simply translational regulation as their absence aids in the detection of foreign nucleic acids. Viruses in contrast may form double-stranded RNA, with the potential addition of 5'-Triphosphate modification during their life cycle. Viruses carrying this 5'-Triphosphate-RNA are recognised and bound by Ifit1, before an Ifit complex (containing Ifit1, Ifit2, and Ifit3) sequester the bound virus for destruction \cite{Pichlmair2011}. 

While little has been published on Ifit1's potential role in hypertension, there is a great deal of potential for linking the two due to the well established link between hypertension and IFN expression. When focusing on the innate arm of immunity in hypertension, it has been established that angiotensin II infusions induce an increase in IFN-$\gamma$ in the spleen and kidney of hypertensive rats \cite{Shao2003}. Studies on IFN's role in inflammation may well provide the link between hypertension and ISGs such as the Ifit family of proteins.

%However, as ISGs in general can be induced by a variety of cellular stressors, it is likely there are broader roles for their expression 

\subsection{Coupling Expression to Cardiovascular Centres of the Brain}

As much of the work of this laboratory is centred around transcriptome analysis of central regions of cardiovascular control there have been terabytes of next-generation sequencing data generated, both published and unpublished. This presented a novel opportunity to reprocess these experimental outputs using the most up to date analysis pipelines as optimised previously. The resulting datasets could be re-mined to assess whether Ifit1 expression was altered centrally between these strains, as it was within the blood. One of the most revealing experimental designs, was the comparison of \acrshort{wky} \textit{versus} \acrshort{shr} at 12 weeks of age. The experiment isolated and sequenced three brain regions that are key in sympathetic and hormonal control of blood pressure, namely the; \acrfull{pvn}, \acrfull{nts}, and \acrfull{rvlm}. Following a re-analysis by way of the optimised pipeline outline above, \acrshort{ifit1} showed a significant upregulation in the \acrshort{shr} \textit{versus} the \acrshort{wky} for both the \acrshort{pvn} and \acrshort{rvlm} (Table \ref{fig:rnaseqifit1_brain}). While the fold change increase was relatively modest compared with the changes observed within blood, the read counts were consistent resulting in a relatively low P-Value. Due to the integrative nature of the \acrshort{pvn}, as outlined in section \ref{brainhypertension}, this region was selected for further validation to assess \acrshort{ifit1} expression. \\

\begin{table}[!htbp]
\centering
\small
\begin{tabular}{llrrrr}
       &           & \multicolumn{4}{c}{DESeq2}                           \\
Region & Gene Name & WKY.AvgCount & SHR.AvgCount & FC          & P-Value  \\
\hline
PVN    & Ifit1     & 23.73        & 67.88     & 1.63  & 2.77E-05 \\
NTS    & Ifit1     & 21.92        & 73.18     & 1.96  & NA       \\
RVLM   & Ifit1     & 14.50        & 52.23     & 1.90  & 7.10E-05
\end{tabular}
\caption[RNAseq generated counts for \acrshort{ifit1} expression in cardiovascular centres of the brain]{RNAseq generated counts for \acrshort{ifit1} expression in cardiovascular centres of the brain. Based on n=3 experiments conducted on \acrshort{wky} \textit{versus} \acrshort{shr} at 12 weeks of age, on cryostat isolated brain regions; \acrfull{pvn}, \acrfull{nts}, and \acrfull{rvlm}. N.B. the "NA" P-Values issued by DESeq2 are returned due to a single sample with an extreme outlier (as detected by Cook's distance).}
\label{fig:rnaseqifit1_brain}
\end{table}


\subsection{\acrfull{facs}}

Up until now this work has focused on \acrshort{ifit1} expression in whole blood samples. However blood itself is comprised of several subpopulations of cell types, each with a different role in physiology. In order to delve further into characterising the overabundance of \acrshort{ifit1} in the hypertensive (\acrshort{shr}) blood the specific subtype of blood cell responsible for its upregulation needed resolving. Of the major blood cell types (i.e. erythrocytes, leukocytes, and thrombocytes), only leukocytes contain a nucleus (nucleated), and are therefore the focus of this study on transcription. Leukocytes as a type can be further split into several different subtypes, each with their own specific role in immunity. 

With this in mind \acrfull{facs} presented an ideal technology for resolving whole blood into its individual constituents, enabling \acrshort{ifit1} expression to be quantified across each subpopulation of leukocytes. Cell sorting takes place based on the presence or absence of specific characteristics, in this case the presence of fluorophores conjugated to antibodies that can bind cell surface protein markers. A combination of excitation lasers and detectors enable the physical separation of cells, depending on how the operator calibrates the machine to isolate cells based on a combination of fluorophore emission (a process known as "Gating"). 

The difficulty presented with this technology, is the optimisation of antibodies to be used and the gating strategy to be employed. With this in mind a literature review revealed a team had optimised and validated a \acrshort{facs} pipeline for rat blood. In 2016 a 9-colour FACS method to characterise major leukocyte populations in the rat was carried out \cite{Barnett-Vanes2016}. Barnett-Vanes \textit{et al.} were able to produce a FACS pipeline that resulted in resolving the key blood leukocyte factions. The protocol made use of a comprehensive gating strategy to resolve the major blood leukocyte populations present in rats (Figure ~\ref{fig:gatingstrat}). In addition to this, the team went on to validate their approach using a model of \acrfull{lps}-induced pulmonary inflammation. The team exposed Sprague-Dawley to \acrshort{lps} (1mg/mL), and collected several tissues including; blood, \acrfull{balf} and lungs. After having carried out a flow-cytometry separation, \acrfull{elisa} was used on \acrshort{balf} tissue to validate their approach. Here the pipeline outlined in Barnett-Vanes \textit{et al.} became a foundation for these experiments. It allowed for the resolution the various cellular populations contained within whole blood for the characterisation of which blood fraction was responsible for the robustly elevated \acrshort{ifit1} expression in the \acrshort{shr} animals. By identifying this fraction it may be possible to guide a future literature review to shed light on Ifit1 and its link towards an immune-based aetiology of hypertension.

\begin{sidewaysfigure}[!htbp]
\centering
\includegraphics[width=0.9\textwidth]{chapter5/19-Jun-2018-Layout_WKY3_1.png}
\caption[Representative gating strategy for 9-Colour FACS Analysis]{Representative gating strategy for 9-Colour FACS Analysis, as taken from Barnett-Vanes \textit{et al} \cite{Barnett-Vanes2016}. Initial gating compares levels of side-scatter (SSC) and forward-scatter (FSC) to identify cells from debris before singlet events are resolved by way of a low Trigger Pulse Width. Single cell events are then classified by viability, as detected by a low signal of the zombie staining method employed. Using the cell surface marker CD45, the leukocyte population can be isolated. The presence of the surface marker CD3 allows the resolution of total T-cells, which can then be separated by the detection of CD4 or CD8 in order to discern Cytotoxic T-cells from T-Helper Cells respectively. From the CD3- population, NK cells can be resolved by the presence of CD161. All remaining cells can be split into B-Cells and non B-cells by the presence of CD45R. A higher level of SSC is indicative of increased internal complexity (i.e. granularity) and allows for the non B-cells to be split into monocytes and neutrophils. Finally, the ratio of His48 and CD43 cell surface markers can be used to separate Classical from Non-classical monocytes.}
\label{fig:gatingstrat}
\end{sidewaysfigure}


\section{Aims}
In this chapter, and building upon the discovery of \acrshort{ifit1} as a potential biomarker, a further characterisation of its expression and involvement in genetically predisposed hypertension will be conducted. This will be achieved via the following;\\
\begin{itemize}
\singlespacing
\setlength
  \item Assess expression within a key cardiovascular centre in the brain \\
	\item Characterise the faction of blood responsible for the elevated expression of ifit1 in the SHR \\
	\item Conducting a literature review to assess role in mechanistic model of hypertension
\end{itemize}


\section{Materials and Methods}

\subsection{Tissue Collection - PVN Isolation}
Rats (WKY Adults, SHR Adults [12 Weeks], WKY Juvenile, and SHR Juvenile [4 Weeks] n=8 per group) were stunned by a blow to the head and then decapitated. A scalpel was used to make a longitudinal incision (rostral-caudal) on the crest of the head, and the scalp was pulled back to reveal the cranium. Two cuts were made, approaching from the site of decapitation to either side of the hindbrain using scissors. This enabled forceps to prise open the top of the cranium, allowing the brain to be gently liberated with a flat spatula. Brains were instantly frozen using powdered dry ice and stored at -80$\degree$C until needed. The brain was sectioned and mapped using a cryostat (Leica Microsystems CM1900 Cryostat). Coronal sections of tissue were cut rostral-caudal at 60$\mu$m thickness and stained with Toluidine blue (0.1\% wt/vol in 70\% EtOH; Sigma Aldrich) in order to map the hypothalamus (Paxinos and Watson Rat Atlas) (Figure ~\ref{fig:pvn}). A micropunch at 1mm diameter (Fine Science Tools) was used to isolate PVN samples in unstained sections and deposit them into a 1.5ml centrifuge tube ready for RNA extraction. \\
	
\begin{figure*}[!htbp]
\centering
\includegraphics[width=0.9\textwidth]{chapter5/PVN_1.png}
\caption{Coronal section of Rat Brain illustrating atlas coordinates and Toluidine Blue stain of the Paraventricular Nucleus (PVN). Here is a coronal section of the rat brain at Bregma -1.8mm showing the location of PVN lateral to the third ventrical. The PVN has been enlarged in the left side of the image to illustrate both a Toluidine Blue stain of the region (Right), and a depiction of how the nuclei are separated (Left; \acrfull{dp}, \acrfull{mn}, \acrfull{mp}, \acrfull{p}, \acrfull{vp}.)}
\label{fig:pvn}
\end{figure*}

\subsection{RNA Extraction}
Similarly to the blood RNA extraction protocol, all procedures for PVN RNA extraction were conducted in an RNAse free environment ensured by good laboratory practice in addition to liberal cleaning of equipment and workspaces with RNAseZap (Ambion). Sample punches were homogenized in 1ml TRIzol reagent (Invitrogen), before centrifuging at $\sim$ 12,000 x g for 12 minutes at 4$\degree$C. Supernatants from each sample were carefully decanted into new tubes, leaving behind a pellet of cellular debris to be discarded. 

One volume of EtOH (100\% v/v) was added directly to one volume of sample homogenate in TRIzol reagent before vortexing to ensure thorough mixing. Then the sample mixtures were added into Zymo-Spin IIC Columns (Zymo Research) placed in collection tubes and centrifuged at $\sim$ 12,000 x g for 1 minute. Spin columns were then transferred to new collection tubes and 400$\mu$l DirectZol RNA PreWash buffer was added to the column. Samples were again centrifuged for 1 minute with the flow through discarded. This step was repeated. Next, 700$\mu$l RNA Wash Buffer was added to the columns and samples were centrifuged for 1 minute. Flow-through was discarded and samples were centrifuged again for 2 minutes to ensure complete removal of the wash buffer. Columns were then transferred to an RNase-free tube and 50$\mu$l of DNase/RNase-free water was added to the spin column. Samples were centrifuged for 1 minute to elute the RNA. Final concentrations of RNA for biomarker validation were confirmed using spectrophotometry (Nanodrop 2000c, Thermo Scientific).

\subsection{cDNA Synthesis}
All cDNA synthesis steps were carried out as outlined in section \ref{RNAQuality&cDNA Synthesis} with the Quantitect RT Kit (Qiagen).

\subsection{Quantitative PCR Analysis}
qPCR Analysis of \acrshort{ifit1} expression in PVN was carried out as outlined in section \ref{qPCR of Target Genes}.

\subsection{9-Colour Panel Antibody Validation}

\subsubsection{Creation of an \acrshort{ifit1} expressing construct}

Initially, RNA was pooled from the SHR juvenile blood samples in order to maximise the levels of \acrshort{ifit1} transcripts contained. A cDNA synthesis was carried out using the Qiagen Quantitect kit, as outlined in section \ref{RNAQuality&cDNA Synthesis}. However, the random hexamer primers supplied with the kit were swapped for an oligo d(T)$\textsubscript{20}$ primer in order to capture full length mRNA transcripts containing a complementary Polyadenylated tail. Once a population of cDNA was produced, PCR Primers were selected to amplify and amend the \acrshort{ifit1} transcript for eventual ligation and transfection (Table ~\ref{tab:PCRPRimers}).

\begin{table}[!htbp]
\centering
\scriptsize
\begin{tabular}{ll}
                          & \\
\textbf{Ifit1 BamH1 (F)}  & 5’ – \textcolor{Aquamarine}{CGC}\textcolor{Plum}{GGATCC}\textcolor{ForestGreen}{GCCACC}\textcolor{Red}{ATG}GGAGAGAATGCTGGTGGTGA\\
\textbf{Ifit1 Xho1 (R)}   & 3’ – \textcolor{Aquamarine}{CCG}\textcolor{BurntOrange}{CTCGAG}GCTGCATTCAAAATGCAGGGTTCAT\\
                          & \\
Annotation;               & \textcolor{Red}{Start Codon}\\
                          & \textcolor{ForestGreen}{Kozak Consensus Sequence}\\
                          & Ifit1 Specific\\
                          & \textcolor{Plum}{BamH1 Digestion Site}\\
                          & \textcolor{BurntOrange}{Xho1 Digestion Site}\\
                          & \textcolor{Aquamarine}{Cleavage Overhang}\\
\end{tabular}
\caption[Primers for PCA amplification of Ifit1 Gene]{Primers for PCA amplification of Ifit1 Gene. Also shown are the specific regions incorporated for later digestion in addition to a Kozak Consensus Sequence for increased transcription.}
\label{tab:PCRPRimers}
\end{table}


Using the oligo d(T)$\textsubscript{20}$ Primer synthesised cDNA as a template, a PCR reaction master mix was made up to 50$\mu$l total (Table ~\ref{tab:PCRMasterMix}). The sample was then ran on a thermocycler to amplify the \acrshort{ifit1} transcript and add on the necessary cleavage sites required for insertion into a vector (Table ~\ref{tab:PCRtimings}). The PCR product was next run on a 1\% Agarose gel to inspect whether it had been successful. A solution of 0.3g Agarose (Sigma Aldrich) was dissolved in 30ml of Millipore water, heated in the microwave, along with 1$\mu$l of Ethidium Bromide (Sigma Aldrich). This was mixed thoroughly and allowed to cool to approximately 50$\degree$C. This mixture was then poured into a small gel rig (Sigma Aldrich) with a loading comb in place and allowed to polymerise. Once polymerised, the gel rig was filled with 1x TAE buffer to cover the gel. 10$\mu$l of Quick-Load Purple 1Kb DNA Ladder (New England Biolabs) was loaded into the first well and two mixtures of 10$\mu$l PCR product, 2$\mu$l 6x Loading Dye (New England Biolabs) and 1$\mu$l Ethidium Bromide (Sigma Aldrich) were added into the adjoining lanes. The gel was run at 125v until the loading dye was approx. 80\% through the gel. Images were taken using the UV transilluminator (Azure Biosystems) to confirm whether a product was seen at all, and which size the band corresponded to. A size band corresponding to the ifit1 gene CDS (1466bp) was indicative of a successful PCR amplification (Figure ~\ref{fig:ifit1PCR}).

\begin{table}[!htbp]
\centering
%\footnotesize
\begin{tabular}{lrr}
  Component                                   & Volume                & Final Concentration     \\
  \hline
  5x Phusion HF Buffer                        & 10$\mu$l              & 1x                      \\
  Primers (Table ~\ref{tab:PCRPRimers})       & 0.2$\mu$ Each         & 200$\mu$M               \\
  10mM dNTPs                                  & 1$\mu$l               & 0.5$\mu$M               \\
  Nuclease-free Water                         & 36$\mu$l              &                         \\
  Phusion DNA Polymerase                      & 0.5$\mu$l             & 1.0 units/50$\mu$l PCR  \\
  cDNA Template & 2$\mu$l & \\
\end{tabular}
\caption{Reaction master mix for Phusion DNA PCR}
\label{tab:PCRMasterMix}
\end{table}

\begin{table}[!htbp]
\centering
%\footnotesize
\begin{tabular}{lrr}
  
  Step                          & Temp                      & Time        \\
  \hline
  Initial Denaturation          & 98$\degree$C              & 30 Seconds  \\
  Amplification (35 Cycles)     & 98$\degree$C              & 1 Minute    \\
                                & 72$\degree$C              & 30 Seconds  \\
  Final Extension               & 72$\degree$C              & 5 Minutes   \\
  Hold                          & 4$\degree$C               & $\infty$    \\
\end{tabular}
\caption{PCR Amplification Protocol}
\label{tab:PCRtimings}
\end{table}

\begin{figure*}[!htbp]
\centering
\includegraphics[width=0.7\textwidth]{chapter5/Ifit1PCR3.png}
\caption{PCR Product of size matching that of Ifit1. Viewed on 1\% Agarose Gel}
\label{fig:ifit1PCR}
\end{figure*}

The two bands were carefully excised from the gel using a UV transilluminator to view the bands and a scalpel blade. Care was taken to remove as much agarose gel as possible. The containing DNA was then isolated using the Gel Extraction kit (Qiagen). The excised gel bands were weighed and 3 volumes of Buffer QC were added to 1 volume of gel (100mg - 100$\mu$l). This solution was incubated at 50$\degree$C for 10 minutes and vortexed every 2 minutes to ensure the agarose gel was full digested. As the adsorption of DNA to the QIAquick membrane is only efficient at acidic conditions of pH $\leq$ 7.5, the Buffer contains a pH indicator. A check of the solution’s colour confirmed the pH to still be acidic, and so no action was required. The QIAquick spin column was then placed in a 2mL collection tube, and the solution was loaded into the column and centrifuged at $\sim$ 14,200 x g for 1 minute. Flow-through was discarded, before 500$\mu$l of Buffer QC was loaded directly onto the column and left for 5 minutes at room temperature. The column was then centrifuged again for 1 minute at $\sim$ 14,200 x g. Next, the column was washed with 75$\mu$l of Buffer PE and left for 3 minutes at room temperature before centrifuging for 1 minute. Flow-through was again discarded, and the column was centrifuged once more for 1 minute to remove any residue. Finally, the column was placed into a clean 1.5mL microcentrifuge tube. To elute the DNA, 30$\mu$l of Buffer EB (10mM Tris-Cl, pH 8.5) was pipetted directly onto the centre of the membrane and left to stand for 1 minute. The sample was then centrifuged at $\sim$ 14,200 x g for 1 minute and DNA was eluted. 

A tube of purified DNA and a pcDNA3.1(+) vector were incubated overnight at 37$\degree$C with a reaction buffer to produce the specific corresponding sticky ends required for ligation of the insert into the vector (Table ~\ref{tab:PCRDigestion}). The digestion products were reran on a 1\% agarose gel and excised, as outlined above, in order to purify the product. In order to optimise the ligation procedure, a concentration calculation was conducted. A 1\% agarose gel was ran with Quick-Load Purple DNA Ladder (New England Biosciences) at varying loading concentrations (2$\mu$l, 4$\mu$l, \& 8$\mu$l). The insert was loaded next at either 2$\mu$l or 4$\mu$l of volume (mixed with 3$\mu$l or 1$\mu$l of water respectively to maintain a loading volume of 5$\mu$l in total). The vector was loaded at either 1$\mu$l or 2$\mu$l of volume (made up to 5$\mu$l with water) in the next two loading wells. This gave the opportunity to make a comparison of concentration against the known concentrations of the maker lanes (Table ~\ref{fig:PCRQuant}). \\

\begin{table}[!htbp]
\centering
\begin{tabular}{lr}
  
  Component                         & Volume\\
  \hline
  cDNA          & 28$\mu$l \\
  XhoI    & 1$\mu$l \\
  BamHI                              & 1$\mu$l \\
  Cut Buffer (10x)               & 5$\mu$l \\
  Water                          & 15$\mu$l  \\
\end{tabular}
\caption{PCR Digestion Mastermix}
\label{tab:PCRDigestion}
\end{table}

\begin{figure*}[!htbp]
\centering
\includegraphics[width=0.9\textwidth]{chapter5/PCR_Quant.png}
\caption{Semi-quantitative assessment of PCR inserts and vector}
\label{fig:PCRQuant}
\end{figure*}



\begin{figure}
\[ x=\frac{kb\ of\ insert}{kb\ of\ vector}\times ng\ of\ vector\]
\normalsize
\begin{align*}
\text{Where;} ~Insert\ length &= \text{1466bp} \\
Vector\ amount &= \text{50ng, as defined by the T4 Ligase Protocol} \\
Vector Length &= \text{5428} \\
\end{align*}
\caption[Equation for calculating Ligation Molar Ratio]{Equation for calculating Ligation Molar Ratio}
\label{eq:ligationmolar}
\end{figure}

Using the concentrations inferred from the gel, a ligation molar ratio could be calculated of insert to vector using the equation outlined in Figure \ref{eq:ligationmolar}. A final quantity of 13.5ng was determined for a 1:1 ratio of Insert:Vector. Since the Promega website state ligations are tested at ratios from 1:1 to 3:1, and given the collective experience of colleagues within the laboratory, a ratio of 3:1 (40.5ng of insert) was selected to ensure an excess of insert without requiring more than 20$\mu$l of reaction volume as outlined in Table ~\ref{tab:Ligation}. This reaction mixture was made up in a microcentrifuge tube on ice and mixed gently by pipetting up and down. It was then incubated at room temperature (21$\degree$C) for 15 minutes before being placed back on ice to chill the reaction mixture.  \\

\begin{table}[!htbp]
\centering
\footnotesize
\begin{tabular}{lr}
  
  Component                         & Volume for 20$\mu$l Reaction\\
  \hline
  T4 DNA Ligase Buffer (10x)        & 2$\mu$l \\
  Vector DNA                        & 50ng (2.5$\mu$l) \\
  Insert DNA                        & 40.5ng (10.8$\mu$l) \\
  Nuclease free water               & to 20$\mu$l \\
  T4 DNA Ligase                     & 1$\mu$l  \\
\end{tabular}
\caption{Ligation Mastermix for T4 DNA Ligase}
\label{tab:Ligation}
\end{table}

\subsubsection{Bacterial Transformation}

The mixture containing ligated and unligated DNA could then be transformed into competent \textit{Escherichia coli} (E. coli) DH5$\alpha$ cells. 50$\mu$l of DH5$\alpha$ in calcium chloride suspension were thawed on ice and mixed with 5$\mu$l of the ligated plasmid. This mixture was incubated on ice for 50 minutes before being placed in a 42$\degree$C water bath for 45 seconds to heat shock the bacteria. This was then placed back on ice for 2 minutes. 450$\mu$l of Super Optimal Broth with glucose (SOC) medium was then added in order to dilute the calcium chloride, stabilising the cells. This was then left in a shaking incubator for 1 hour at 37$\degree$C. 100$\mu$l of the cells were plated onto an agar plate containing ampicillin and incubated at 37$\degree$C overnight. Using a pipette tip, three colonies were isolated and grown overnight in a 37$\degree$C shaking incubator in 5ml of Lysogeny Broth (LB) containing ampicillin. The resulting bacterial suspensions were passed through a QIAprep\textsuperscript{\textregistered} Spin Miniprep Kit (Qiagen) to isolate plasmid DNA. 1ml of bacterial culture was pelleted by centrifugation at $>$6,800 x g for 3 minutes at room temperature (15-25$\degree$C). The pellets were next resuspended in 250$\mu$l of Buffer P1 and transferred to a microcentrifuge tube. 250$\mu$l of Buffer P2 was then added and the mixture was inverted thoroughly until the solution became clear. Once the solution was clear, 350$\mu$l of Buffer N3 was added and the mixtures were immediately mixed thoroughly before centrifuging for 10 minutes at 17,900 x g. The supernatant from the previous step was decanted via a pipette to a QIAprep spin column and centrifuged for 30-60 seconds. The flow-through was discarded. via an additional centrifugation, 0.75ml of Buffer PE was used to wash through the spin columns and again the flow-through was discarded. The columns were centrifuged at the above speed for 1 minute in order to remove residual wash buffer. Next, the columns were transferred to a clean 1.5ml microcentrifuge tube and 50$\mu$l of Buffer EM (10mM Tris$\cdot$Cl) was added to the centre of the column. These were left to stand for 1 minute before centrifuging for 1 minute to elute the plasmid DNA. A nanodrop spectrophotometer was used to quantify the resulting plasmid DNA (Table ~\ref{tab:construct_nanodrop}). \\

\begin{table}[!htbp]
\centering
\footnotesize
\begin{tabular}{lrrrrr}
  
  Colony    & Conc (ng/$\mu$l)  & A260 & A280 & 260:230 & 260:230 \\
  \hline
  A         & 205.7             & 4.11 & 2.23 & 1.84    & 1.83  \\
  B         & 276.4             & 5.53 & 2.99 & 1.85    & 1.78  \\
  C         & 347.5             & 6.95 & 3.72 & 1.87    & 1.97  \\
\end{tabular}
\caption{Spectrophotometry output for Isolated Ifit1 constructs}
\label{tab:construct_nanodrop}
\end{table}

As the pcDNA3.1($+$) vector confers ampicillin resistance, growth of colonies on the ampicillin containing agar only confirmed successful transformation of the plasmid. These may contain the insert of interest, or may well be re-ligated vectors. In order to confirm a successful ligation of the vector and insert, the plasmid DNA was initially incubated with BamHI and XhoI and ran on a 1\% agarose gel to assess the size of bands produced from the digestion (Table ~\ref{tab:construct_digestion}, Figure ~\ref{tab:construct_digestion}). Three distinct bands are apparent from the BamHI and XhoI digestion at; ~5,500bp, ~3,500bp and ~1,400bp. The whole pcDNA3.1($+$) is 5428bp in size, which seems consistent with the strongest band at ~5,500bp. This band appears to be either undigested, or simply does not contain the insert and so digestion only serves to linearise the plasmid as the BamHI and XhoI sites are only 56bp away from each other. The band at ~1,500 appears to be the Ifit1 insert itself, as the insert is 1466bp and directly flanked by BamHI and XhoI digestion sites. \\

\begin{figure*}[!htbp]
\centering
\includegraphics[width=0.9\textwidth]{chapter5/PlasmidDigestion.png}
\caption{BamHI \& XhoI Digest on Plasmid Construct}
\label{fig:plasmid_digest}
\end{figure*}

\begin{table}[h]
\centering
\footnotesize
\begin{tabular}{lr}
  
  Component     & Volume per Reaction ($\mu$l) \\
  \hline
  BamHI         & 0.5  \\
  XhoI          & 0.5  \\
  Plasmid cDNA  & 5  \\
  Cut Buffer (10x) & 5  \\
  Water         & 3  \\
\end{tabular}
\caption{Digestion Mastermix for confirming insert in plasmid construct}
\label{tab:construct_digestion}
\end{table}

For additional confirmation, 1500ng of each sample DNA was sent for Sanger Sequencing with primers corresponding to either side of the ligation sites (Table ~\ref{tab:SangerPrimerSequences}). The results were curated and both ends of the sequencing were merged to give a final sequence of the inserts. These were then ran through a BLAST search algorithm to confirm the sequence identity with the NCBI Ifit1 Rattus norvegicus sequence. Each of the inserts showed a high sequence identity with Ifit1 sequence entry and so were deemed suitable for usage downstream (Appendix \ref{blast}).  \\

\begin{table}[!htbp]
\footnotesize
\centering
\begin{tabular}{lr}
Primer Site & Sequence \\
\hline
\small{T7 Promoter} & {\textit{5’-TAATACGACTCACTATAGGG-3’}} \\
\small{pCR3.1-BGHrev} & {\textit{5’-TAGAAGGCACAGTCGAGG-3’}} \\
\end{tabular}
\caption{Sanger Sequencing of Plasmid Primer Sequences}
\label{tab:SangerPrimerSequences}	
\end{table}

\subsubsection{Transfection into HEK293t Cell line}

Six wells of a 24-well plate were seeded with carefully thawed Human Embryonic Kidney Cells, expressing a mutant version of the SV40 large T antigen (HEK293T). They were maintained in Dulbecco's Modified Eagle Medium containing pH indicator, with additional L-Glutamine (4mM), Streptomycin (1\%), and Fetal Bovine Serum (FBS, 10\% Thermofisher). These were maintained with fresh medium and incubated at 37$\degree$C until they reached approximately 70\% confluency. At this point, a lipofectamine transfection could be carried out of either control plasmid (pcDNA3.1($+$)) or the plasmid construct containing rat Ifit1 (pcDNA3.1($+$)$\cdot$Ifit1) as per the manufacturer's instructions (Invitrogen) \cite{Invitrogen}. In two 1.5ml microcentrifuge tube, 4.5$\mu$l Lipofectamine 3000 Reagent was diluted in 150$\mu$l Opti-MEM Reduced Serum Medium and mixed well. In two additional tubes, 150$\mu$l of Opti-MEM media was mixed with 1.5$\mu$g of respective DNA (1.5$\mu$l pcDNA3.1($+$), 3.78$\mu$l pcDNA3.1($+$)$\cdot$Ifit1). To both of these, 3$\mu$l of p3000 Reagent was added. Each of the tubes were combined with one of the previously prepared Lipofectamine tubes and left to incubate at room temperature for 5 minutes. Serum containing media was decanted from each of the wells on the plate, and replaced with Opti-MEM Reduced serum medium before dividing each lipofectamine mixture between three wells (3 x pcDNA3.1($+$), 3 x pcDNA3.1($+$)$\cdot$Ifit1) and incubating at 37$\degree$C. Cells were checked several times daily to prevent toxically high confluence from occurring. After 2-days of expression, cells could be lysed to confirm protein expression.

\subsubsection{Confirmation of Protein Expression}

Transfected cells (HEK293T Control and HEK293T$\cdot$Ifit1) were lysed on their respective cell culture dishes using 150$\mu$l of lysis solution consisting of Sodium dodecyl sulfate (SDS), Tris$\cdot$HCl and HEPES$\cdot$NaOH. After the solution was applied to each of the plates, a cell scraper (Sigma-Aldrich) was used to detach the cells from the plate and ensure thorough lysis. Cell lysates were collected into a 1.5ml centrifuge tube, and passed through a 5mL syringe with 25G needle to break up the cells further. 
	
Protein concentrations of both lysates was measured using the Nanodrop 2000c (Thermofisher). This was calculated by assessing A280 absorbance values with 1 Abs being equal to 1mg/mL. Protein concentrations were relatively equal and therefore did not warrant further dilutions to normalise between the two groups. 

Next, 50$\mu$l of 4x loading buffer (Table ~\ref{tab:WBBuffers}) was added to each of the sample lysates to make loading of the samples into their respective wells easier, as well as enabling visualisation of the bands as they progress through the electrophoresis gel. Sample lysates were next heated on a heat block to 95$\degree$C briefly to ensure any SDS had not precipitated in the lysate mixes and the proteins were properly denatured prior to gel separation. 

A 10\% electrophoresis gel was prepared (Table ~\ref{tab:WBBuffers}), with a 4\% stacking gel component. This was prepared along with 1L of running buffer. 

Along with the HEK293T and HEK293T$\cdot$Ifit1 sample lysates, lysates from two other cell lines were included to assess endogenous expression of ifit1 for potential involvement in further validation studies. These were the Att20 mouse pituitary cells and the PC12 rat adrenal medulla cells and were available due to ongoing experiments of colleagues within the laboratory. All four cell lysates were loaded at both 30$\mu$g and 60$\mu$g concentrations, as well as 10$\mu$l of protein size ladder (Novex™ Sharp Pre-Stained Protein Standard, Thermofisher). 

The gel was run for approximately 45 minutes at 180v, until the size markers had separated sufficiently to easily resolve the 50kDa marker. At this point a semi-dry protein transfer was conducted using the Trans-Blot SD Semi-Dry Transfer Cell (Bio-Rad). A transfer buffer was initially prepared (Table ~\ref{tab:WBBuffers}). A PVDF membrane (Millipore) was first activated in 100\% Methanol for 30 seconds before rinsing with dH2O and submerging in transfer buffer to acclimatise prior to the transfer. Four sheets of filter paper were used to sandwich the PVDF membrane and electrophoresis gel. The stack was soaked in transfer buffer and ran at 20v for 110 minutes. The PVDF membrane was soaked in Ponceau Red reagent (0.1\% w/v Ponceau, 5\% v/v Acetic acid) for approximately 10 minutes then rinsed in dH$\textsubscript{2}$O to visualise the protein transfer was successful. The membrane was destained with several washes in \acrfull{bsa} in \acrfull{pbs} (5\% w/v) ready for antibody visualisation (Figure ~\ref{fig:antibody_validation}).


\begin{table}[!htbp]
\centering
\footnotesize
\begin{tabular}{lrr}
  
  Buffer                        & Constituents                        & Quantity        \\
  \hline
  2x Separation Buffer          & Tris Base                           & 9.08g  \\
                                & SDS                                 & 0.2g  \\
                                & HCl/NaOH                            & pH to 8.8\\
                                & H$\textsubscript{2}$O               & Up to 100mL \\
  \hline
  2x Stacking Buffer            & Tris Base                           & 3g  \\
                                & SDS                                 & 0.2g  \\
                                & HCl/NaOH                            & pH to 6.8 \\
                                & H$\textsubscript{2}$O               & Up to 100mL \\
  \hline
  1x Separation Gel (10\%)      & 2x Separation Buffer                & 5mL  \\
                                & BisAcrylamide (40\%)                & 2.5mL  \\
                                & H$\textsubscript{2}$O               & 2.5mL  \\
                                & TEMED                               & 20$\mu$L  \\
                                & APS (10\% in H$\textsubscript{2}$O) & 100$\mu$L  \\
  \hline
  1x Stacking Gel (4\%)         & 2x Stacking Buffer                  & 1mL  \\
                                & BisAcrylamide (40\%)                & 170$\mu$L  \\
                                & H$\textsubscript{2}$O               & 824$\mu$L  \\
                                & TEMED                               & 4$\mu$L  \\
                                & APS (10\% in H$\textsubscript{2}$O) & 20$\mu$L  \\
  \hline
  10x Running Buffer            & Tris Base                           & 30.3g  \\
                                & Glycine                             & 144g  \\
                                & SDS                                 & 10g    \\
                                & HCl/NaOH                            & pH to 8.3 \\
                                & H$\textsubscript{2}$O               & Up to 1L  \\
  \hline
  1x Transfer Buffer            & Tris Base                           & 3g  \\
                                & Glycine                             & 15g  \\
                                & Methanol                            & 200mL    \\
                                & HCl/NaOH                            & pH to 8.3 \\
                                & H$\textsubscript{2}$O               & Up to 1L  \\
  \hline
  4x Loading Buffer             & 1M Tris HCl                         & 2mL \\
                                & SDS                                 & 0.8g \\
                                & Glycerol                            & 4mL \\
                                & 0.5M EDTA                           & 1mL \\
                                & Bromophenol Blue                    & 8mg \\
                                & 2M Dithiothreitol                   & 2mL \\
                                & HCl/NaOH                            & pH to 6.8 \\
\end{tabular}
\caption{Reagents used for Western Blot analysis of Ifit1 protein expression}
\label{tab:WBBuffers}
\end{table}

\begin{figure*}[!htbp]
\centering
\includegraphics[width=0.9\textwidth]{chapter5/westernblot.png}
\caption[Antibody test on various cell lysates]{Antibody test on various cell lysates. Here Hek293t cells, either transfected with Ifit1 or a control pcDNA3.1 construct, were lysed and ran on a Western Blot before being incubated with the Anti-Ifit1 antibody. Also featured are cell lysates from Att20, and PC12 cells. Two clear bands can be seen just above the 55kDa size, a size consistent with the Ifit1 protein. There also appears to be a distinct band in both of the Att20 lysates at approximately 100kDa in size. This may be a product of ifit protein dimerisation, due to insufficient SDS denaturation. Indeed the IFIT2 protein has been shown to dimerise \textit{in vivo} \cite{Yang2012}. However the evidence for IFIT1 dimerisation is less clear. Further characterisation of these bands would be possible through extraction and interrogation using Mass Spectrometry techniques.}
\label{fig:antibody_validation}
\end{figure*}

\subsubsection{Immunocytochemistry}

A newly transfected population of cells (as outlined above) were grown on glass cover slips (Thermanox - Nalge Nunc International, NY, USA). Growth media was aspirated from them before being gently washed with 1mL of sterile \acrshort{pbs}. This was carefully removed, and 1mL 4\% Paraformaldehyde (PFA) was added to fix the cells. This was left on for 15 minutes at room temperature. Following fixation, cells were washed a further two times with fresh \acrshort{pbs}. Cells could then be blocked and permeabilised by incubation for 10 minutes with permeabilisation buffer (2\% Bovine Serum Albumin (\acrshort{bsa}), 0.1\% Triton X-100). Primary Anti-Ifit1 diluted in \acrshort{pbs} containing 5\% \acrshort{bsa} was dropped onto parafilm strips in a light-sealed box. Coverslips could then be removed from the cell culture plates and inverted ontop of the parafilm to enable an economical usage of primary antibody. Coverslips were light-sealed and left overnight at 4$\degree$C before being washed with PBS three times. Using the same inverted incubation method coverslips were incubated with secondary antibodies diluted in PBS with 5\% BSA for 1 hour at room temperature in the dark. After this incubation period DAPI was added and samples were left for a further 10 minutes. Following three additional washes in BSA samples were imaged on a DMI6000B microscope (Figure \ref{fig:antibody_immunocyto}; Leica - Wetzlar, Germany). \\

\begin{figure*}[!htbp]
\centering
\includegraphics[width=1\textwidth]{chapter5/immunocytochemistry.pdf}
\caption[Antibody test on transfected HEK293T cells]{Antibody test on HEK293T cells containing either control transfection (pcDNA3.1(+)) or Ifit1 transfection (pcDNA3.1(+)Ifit1). Cells were incubated with anti-Ifit1 antibody (1:50, 500ms exposure, red) and DAPI (1:1000, 1-20s exposure, blue) before being imaged at 10x and 40x magnification. Here, the ifit1 transfected cells show a clear signal in the red channel whereas no signal can be seen in the control transfected cells.}
\label{fig:antibody_immunocyto}
\end{figure*}


\subsection{Fluorescence-activated cell sorting}

\subsubsection{Antibody Panel Viability}

Once antibodies had been delivered, the first step was to assess their integrity was not compromised during transit, and that their binding efficiency was suitable for my intended downstream application. To do this, 2$\mu$l of each antibody was added to a glass tube containing ~1 drop of OneComp eBeads™ Compensation Beads (ThermoFisher Scientific). Each drop of beads contains two populations; a positive population that has reactivity with any rat, mouse or hamster antibody, and a negative population that has no reactivity towards any antibodies. Firstly, a solution of unstained beads was ran through the cytometer to show the FSC/SSC of 10,000 events. This allowed a single-cell gate to be drawn across the relevant proportion of events prioritised for antibody optimisation, preventing any cohered beads from passing through at the same time and flagging a false positive/negative. Then each antibody containing tube was ran individually through the FACS machine (BD Fortessa X20 analyser, BD Biosciences - Franklin Lakes, NJ USA) with their corresponding excitation wavelength laser and filter parameters set up to a measured total of 10,000 events (Table ~\ref{tab:9-colourPanel}). 
        
The resulting emission profiles showed a bimodal distribution for each antibody, with a clear separation observed between the negative and positive peaks. Care was taken to ensure each peak lay on a linear, and not logarithmic, measure apart. This keeps the final measurements within the dynamic range of the antibodies, allowing quantities to be measured more accurately than if binding was outside of the linear range. This process also enables a pragmatic test as to whether there is any cause for concern with the panel design by highlighting any potential crossover of emission bands. Based on the antibody emissions from this optimisation, the software can mitigate this by creating compensation parameters to better resolve peaks that show similar profiles.

\begin{sidewaystable}[!htbp]
\centering
\footnotesize
\begin{tabular}{lrrrrrrrr}
          &                       &                 &                 &             & Suggested & Optimised &             &           \\
Antibody  & Indicates             & Fluorochrome    & Supplier        & Clone       &  Dilution &  Dilution & Excitation  & Emission  \\
\hline
Live-Dead & Cell Viability        & APC-Cy7         & Biolegend       & NIR Zombie  & 1:1000    & 1:1000    & 633nm       & 746nm \\
His48     & Neutrophil            & FITC            & BD              & HIS48       & N/A       & 1:400     & 488nm       & 520nm \\
          &                       &                 & Pharminogen     &             &           &           &             &       \\
CD43      & Monocytes             & PE              & Biolegend       & W3/13       & 1:100     & 1:1000    & 496nm       & 578nm \\
CD161     & NK Cells              & APC             & Biolegend       & 3.2.3       & 1:1000    & 1:200     & 633nm       & 650nm \\
CD4       & T-Helper              & V450            & BD              & OX-35       & 1:10-50   & 1:100     & 404nm       & 448nm \\
          & Cells                 &                 & Biosciences     &             &           &           &             &       \\
CD8a      & T-Cytotoxic           & PerCP-eF710     & Biolegend       & OX-8        & 1:20      & 1:75      & 482nm       & 675nm \\
          & Cells                 &                 &                 &             &           &           &             &       \\
CD3       & T-Cells               & APC Vio770      & Miltenyi        & REA223      & 1:10      & 1:10      & 652nm       & 776nm \\
          & (All)                 &                 & Biotec          &             &           &           &             &       \\
CD45R     & B-Cells               & PC7             & eBioscience     & HIS24       & N/A       & 1:160     & 488-561nm   & 775nm \\
CD32      & Fc$\gamma$II          & N/A             & BD              & D34-485     & 1:100     & 1:50      & N/A         & N/A   \\
          &                       &                 & Pharminogen     &             &           &           &             &       \\
CD45      & B-Cells               & AlexaFluor 700  & Biolegend       & OX-1        &           & 1:50      & 696nm       & 719nm \\
\end{tabular}
\caption{Panel design for FACS isolation of various Leukocyte populations in rat whole blood}
\label{tab:9-colourPanel}
\end{sidewaystable}

\subsubsection{Tissue Isolation}

Following a week habituation period, 4 WKY and 5 SHRs at 12 weeks old were sacrificed by a cranium strike stun and guillotine. Whole blood was collected into Sodium Citrate containing tubes (BD Vacutainer) on ice to prevent coagulation and RNA degradation respectively. To obtain the plasma layer, blood was transferred into 2mL microcentrifuge tubes and spun at 1300 x g for 10 minutes at 4$\degree$C. The upper plasma layer was removed and frozen at -80$\degree$C for further experiments. The next step was to perform a red blood cell (RBC) lysis preparation to deplete the samples from erythrocytes that could interfere with the dynamic range of the FACS experiment. This was carried out using an ammonium chloride based Lysing Buffer (BD Biosciences) as per the manufacturer’s recommended protocol. For every 200$\mu$l of whole blood, 2.0mL of lysing solution was added. Tubes were gently vortexed immediately after adding the lysing solution, and incubated at room temperature for 15 minutes in an area out of the light. Once samples were no longer cloudy, they were centrifuged for 5 minutes at 200 x g. The supernatant was carefully aspired without disturbing the pellet, and discarded. FACS buffer was made up for diluting antibodies and consisted of 0.5\% \acrshort{bsa} (Sigma-Aldrich) in \acrshort{pbs} (Sigma-Aldrich). This solution was passed through Millipore filters in order to sterilise it. The now RBC depleted pellet was resuspended in FACS buffer for flow cytometric analysis. 

In order to assess both the success of the RBC depletion, and calculate how many leukocytes are present in the final FACS suspension, hemocytometry was conducted. The population of leukocytes was obtained, suspended in FACS buffer and integrity was verified, antibodies could be added to aid in identification and separation of the various cellular factions present. This was initially done by taking a 100$\mu$l aliquot of the undiluted RBC depleted FACS suspension and mixing with 100$\mu$l of Trypan Blue Solution (0.4\%, Sigma) for identification of non-viable cells that have ruptured. A cover slip was placed on top of the Bright-Line™ hemocytometer (Sigma-Aldrich) and aligned so it rested balanced on the two spacing risers either side of the count matrices. 50-100$\mu$l of the FACS suspension/Trypan Blue mix was pipetted onto both of the counting matrices to create a drop of the stained suspension on each. Using a light microscope at 100x magnification (Nikon Eclipse TS100) and a hand counter, I was able to count cells within each of the five large 5x5 grids. Cells were excluded if they were touching the upper or left boundaries of the matrix and care was taken to discern between trypan blue stained cells (ruptured) and non-stained cells in the final counts. Then using the below formula, a calculation of viable cells per mL was made (Figure \ref{cellcount}).

\begin{figure}[!hbtp]
\[ Cell\ Viability =\frac{Number\ of\ Live\ Cells}{Number\ of\ Dead Cells}\] \\
\[ Measured\ Cell\ Density\ (Cells/mL) =\frac{Average\ Cells\ per\ Square \times Dilution\ Factor}{Volume\ of\ a\ small\ square\ (mL)}\] \\
\[ Total\ Viable\ Cells = Cell\ Density\ (Cells/mL) \times Volume\ (mL) \times Viability\] \\
\[ Final\ Volume\ (mL) =\frac{Total\ Viable\ Cells}{Target\ Density\ (Cells/mL)}\] \\
\caption[Calculations for Cell Abundance and Viability]{Calculations for Cell Abundance and Viability}
\label{cellcount}
\end{figure}


\subsubsection{Optimisation of Antibody Dilutions}

Before carrying the final FACS sorting experiment, it was pertinent to optimise the antibody dilutions so as to ensure sufficient antibody is present and to minimise non-specific binding by excess probing. To do this, one of the samples obtained from the SHR blood was selected for an antibody titration. A serial dilution of each antibody in FACS buffer was first made with the recommended dilution being used as the starting concentration. The dilutions tested are listed in Table ~\ref{tab:9-colourPanel}.

Between 200,000-500,000 cells used for each antibody dilution. Cells were pelleted by spinning at 200g for 5mins and supernatant was removed, whilst being careful not to dislodge the cell pellet. 25$\mu$l of Anti-CD32 antibody was added at a concentration of 1:50 to the cell pellet and mixed. This was done in order to block antibodies binding non-specifically to the Fc$\gamma$II receptor. The solution was incubated for 10 minutes at room temperature. Following this incubation, 25$\mu$l of each antibody dilution (in FACs buffer) was added to each respective tube containing Fc$\gamma$II-blocked cells. These were briefly mixed by vortexing before incubated on ice, and in the dark, for 30 minutes. Cells were washed with 500$\mu$l of FACs buffer and vortexed before re-spinning to once again pellet the cells. Supernatants were discarded carefully, and cells were resuspended in 300$\mu$l of FACs buffer ready for cytometry assessment of antibody concentrations.

The suitability of each antibody dilution was mathematically assessed by use of the stain index (Figure ~\ref{eq:si}, Table ~\ref{tab:MFI_Optimisation}). The highest stain index denotes the concentration most able to discern between an antigen positive and negative population. For this reason, efforts were made to pick dilutions that would give a suitably high stain index, whilst ensuring an economical usage of the antibody. The antibody dilutions selected are highlighted in Table ~\ref{tab:MFI_Optimisation} in bold. \\

\begin{figure}[!htbp]
\[ Stain\ Index=\frac{[MFI_1 - MFI_2]}{2\sigma}\]
\normalsize
\begin{align*}
\text{Where;} ~ MFI&= \text{Mean\ Fluorescence\ Intensity} \\
\sigma &= \text{Standard\ Deviation\ of\ Negative\ MFI} \\
\end{align*}
\caption[Equation to calculate the Stain Index for Fluorescently conjugated Antibodies]{Equation to calculate the Stain Index for Fluorescently conjugated Antibodies}
\label{eq:si}
\end{figure}

\begin{sidewaystable}[!htbp]
\scriptsize
\centering
\begin{tabular}{llrrrr}
               &  & Mean & Mean & Mean  &  \\
               &  & Fluorescence &  Fluorescence  &  Fluorescence & Staining  \\
Antibody               & Dilution & Intensity  (Positive) &Intensity  (Negative) &  Intensity  (Negative) SD &  Index \\
\hline
\multirow{5}{*}{CD4}   & \textbf{1:100}    & \textbf{2476}                                    & \textbf{121}                                     & \textbf{78.3}                                       & \textbf{15.04}          \\
                       & 1:200    & 1579                                    & 113                                     & 77.2                                       & 9.49           \\
                       & 1:400    & 998                                     & 139                                     & 96.6                                       & 4.45           \\
                       & 1:800    & 682                                     & 148                                     & 89.2                                       & 2.99           \\
                       & 1:1600   & 1034                                    & 173                                     & 104                                        & 4.14           \\
\hline
\multirow{5}{*}{CD45}  & \textbf{1:50}     & \textbf{3448}                                    & \textbf{55.2}                                    & \textbf{126}                                        & \textbf{13.46}          \\
                       & \textbf{1:100}    & \textbf{2442}                                    & \textbf{62.9}                                    & \textbf{127}                                        & \textbf{9.37}           \\
                       & 1:200    & 1573                                    & 57.8                                    & 126                                        & 6.01           \\
                       & 1:400    & 946                                     & 62.9                                    & 116                                        & 3.81           \\
                       & 1:800    & 642                                     & 199                                     & 144                                        & 1.54           \\
\hline
\multirow{4}{*}{CD3}   & \textbf{1:10}     & \textbf{500}                                     & \textbf{8.98}                                    & \textbf{21.4}                                       & \textbf{11.47}          \\
                       & 1:20     & 453                                     & 8.98                                    & 20.8                                       & 10.67          \\
                       & 1:40     & 230                                     & 6.42                                    & 19.2                                       & 5.82           \\
                       & 1:80     & 127                                     & 5.13                                    & 17                                         & 3.58           \\
\hline
\multirow{4}{*}{CD8a}  & 1:300    & 8318                                    & 285                                     & 537                                        & 7.48           \\
                       & \textbf{1:600}    & \textbf{8524}                                    & \textbf{275}                                     & \textbf{541}                                        & \textbf{7.62}           \\
                       & 1:1200   & 2136                                    & 245                                     & 484                                        & 1.95           \\
                       & 1:2400   & 1835                                    & 266                                     & 460                                        & 1.71           \\
\hline
\multirow{3}{*}{CD43}  & \textbf{1:1000}   & \textbf{4167}                                    & \textbf{166}                                     & \textbf{179}                                        & \textbf{11.18}          \\
                       & 1:2000   & 1998                                    & 146                                     & 131                                        & 7.07           \\
                       & 1:3000   & 1751                                    & 139                                     & 122                                        & 6.61           \\
\hline
\multirow{5}{*}{CD45R} & 5:200    & 13144                                   & 136                                     & 162                                        & 40.15          \\
                       & 5:400    & 7629                                    & 88.7                                    & 118                                        & 31.95          \\
                       & \textbf{5:800}    & \textbf{3769}                                    & \textbf{53.9}                                    & \textbf{82.6}                                       & \textbf{22.49}          \\
                       & 5:1600   & 2253                                    & 38.5                                    & 67.8                                       & 16.33          \\
                       & 5:3200   & 1012                                    & 29.5                                    & 53.2                                       & 9.23           \\
\hline
\multirow{5}{*}{CD161} & 1:100    & 1784                                    & 37.2                                    & 66.9                                       & 13.06          \\
                       & \textbf{1:200}    & \textbf{1748}                                   & \textbf{28.2}                                    & \textbf{59.1}                                       & \textbf{14.55}          \\
                       & 1:400    & 1680                                    & 23.1                                    & 54.1                                       & 15.31          \\
                       & 1:800    & 1558                                    & 21.8                                    & 53.8                                       & 14.28          \\
                       & 1:1600   & 1174                                    & 21.8                                    & 51.8                                       & 11.12          \\
\hline
\multirow{5}{*}{His48} & 1:100    & 4623                                    & 46.2                                    & 54.2                                       & 42.22          \\
                       & 1:200    & 7529                                    & 43.6                                    & 54                                         & 69.31          \\
                       & \textbf{1:400}    & \textbf{6896}                                    & \textbf{43.6}                                    & \textbf{51.3}                                       & \textbf{66.79}          \\
                       & 1:800    & 4416                                    & 43.6                                    & 53.4                                       & 40.94          \\
                       & 1:1600   & 2707                                    & 44.9                                    & 55.9                                       & 23.81         
\end{tabular}
\caption{Optimisation of Antibody Dilutions via the calculation of a Stain Index for each}
\label{tab:MFI_Optimisation}
\end{sidewaystable}


\subsection{FACS Analysis on Permeabilised Cells}

\subsubsection{Validation of Biotinylated Anti-Ifit1 Antibody} \label{ValidationBiotinylatedAnti-Ifit1}

In order to attempt a positive selection of Ifit1 expressing blood cells, and to ensure it could be incorporated into the 9-panel \acrshort{facs} protocol, a biotinylated form of the same anti-Ifit1 clone was employed. To validate its suitability in a \acrshort{facs} protocol, the Control (pcDNA3.1) and Ifit1 (pcDNA3.1($+$)$\cdot$Ifit1) transfected cells were obtained via the above protocol. Initially, \acrshort{facs} Buffer (0.5\% \acrshort{bsa} in \acrshort{pbs}), Fixation Buffer (4\% PFA in \acrshort{pbs}), and Permeabilisation Buffer (0.1\% Triton X-100, 0.5\% \acrshort{bsa} in \acrshort{pbs}) were all made up using autoclaved \acrshort{pbs} solution. 

\begin{figure*}[!htbp]
\centering
\includegraphics[width=0.9\textwidth]{chapter5/800_400.png}
\caption[FACS Validation of Anti-Ifit1 Dilutions (1:800 \& 1:400)]{FACS Validation of Dilutions for Biotinylated Anti-Ifit1 Antibody}
\label{fig:FACS_Biotin_Ifit1_Validation}
\end{figure*}

\begin{figure*}[!htbp]
\ContinuedFloat 
\centering
\includegraphics[width=0.9\textwidth]{chapter5/200_100.png}
\caption[FACS Validation of Anti-Ifit1 Dilutions (1:200 \& 1:100)]{FACS Validation of Dilutions for Biotinylated Anti-Ifit1 Antibody}
\label{fig:FACS_Biotin_Ifit1_Validation}
\end{figure*}

\begin{figure*}[!htbp]
\ContinuedFloat 
\centering
\includegraphics[width=0.9\textwidth]{chapter5/50.png}
\caption[FACS Validation of Anti-Ifit1 Dilutions (1:50)]{FACS Validation of Dilutions for Biotinylated Anti-Ifit1 Antibody}
\label{fig:FACS_Biotin_Ifit1_Validation}
\end{figure*}

After 48 hours since transfection, cells were Trypsin-detached (0.25\%, sterile-filtered, BioReagent 2.5 g porcine trypsin and 0.2 g EDTA, Sigma-Aldrich) and pelleted at 180 x g for 3 minutes. These populations were washed with 1mL of sterile \acrshort{pbs} and pelleted again. This wash step was repeated. Cells were counted using a hematocytometer as outlined above and resuspended at approx. 1x10$\textsuperscript{6}$ Cells/100$\mu$l \acrshort{pbs}. At this point the cells could be incubated with 1$\mu$l of Zombie dye per 100$\mu$l of cells for 15-30 minutes at room temperature in the dark. They were then washed with 2mL of \acrshort{facs} Buffer and resuspended to approx 3.24x10$\textsuperscript{6}$ Cells/mL in ice cold \acrshort{facs} buffer. The following fixation method was adapted from Abcam's \acrshort{facs} Protocol database and conducted in the dark to preserve fluorescence of the Zombie Stain \cite{Abcam}. Cells were pelleted at 100 x g for 2 minutes before being resuspended in Fixation Buffer for 10-15 minutes. Cells were then washed with 2mL of Permeabilisation Buffer for 15 minutes and centrifuged at 100 x g for 5 minutes. The supernatant was discarded and they were resuspended in 2mL of 1\% \acrshort{bsa} in \acrshort{pbs}. The Ifit1 transfected cells were divided into 5 groups for the assessing the following antibody dilutions; 1:50, 1:100 (Recommended), 1:200, 1:400, 1:800 (Figure \ref{fig:FACS_Biotin_Ifit1_Validation}). The control transfected cells were divided into 10 groups, as a no-primary antibody control was to be used for each dilution. Primary biotinylated anti-Ifit1 antibody was added to each of the populations at the above dilutions, apart from the no-primary controls. The groups were left to incubate for 1 hour at room temperature. Samples were then washed with incubation buffer (1\% \acrshort{bsa} in \acrshort{pbs}) and centrifuged at 100 x g for 3 minutes to pellet. The supernatant was carefully aspirated and discarded, and the wash step was repeated. Cell pellets were then resuspended in 100$\mu$l streptadvidin conjugated secondary, diluted 1:400 in 0.5\% \acrshort{bsa} containing \acrshort{pbs} (BV605 Streptavidin, BD Horizon). These were then left for 30 minutes incubating at room temperature in the dark. Samples were washed twice in incubation buffer as outlined above before being stored at 4$\degree$C in 0.5\% \acrshort{bsa} \acrshort{pbs} with 1mM EDTA and 0.1\% Sodium Azide ready for \acrshort{facs} analysis. 

\section{Results}

\subsection{Ifit1 Expression in the PVN}

Figure \ref{fig:qpcrIfit1PVN} shows the relative expression of Ifit1 in punched \acrshort{pvn} sections between the \acrshort{wky} and \acrshort{shr}, as both juvenile and adult ages. Similarly to within blood tissue, Ifit1 shows a marked increase in abundance in the \acrshort{shr} as compared with the \acrshort{wky}, across both ages tested (Figure ~\ref{fig:qpcrIfit1PVN}; Juvenile WKY vs SHR, +10.1 Fold, $\textit{P-Value$<$0.0001}$; Adult WKY vs SHR, +15.5 Fold, $\textit{P-Value=0.0002}$). 

\begin{figure}[!htbp]
  \centering 
  \begin{tabular}{c}
  \includegraphics[width=0.6\textwidth]{chapter5/Ifit1_pvn.pdf} \\
\end{tabular} 
  \caption[qPCR Relative Expression of Ifit1 in the PVN]{qPCR Relative Expression of Ifit1 in the PVN, displayed as $2^{-\Delta\Delta C_{T}}$ values. Statistical analysis made use of a Student's T-Test (P-Value $<$0.05, *; $<$0.01, **; $<$0.001, ***;  $<$0.0001, ****).}
  \label{fig:qpcrIfit1PVN}
\end{figure}



\subsection{FACS Analysis to Isolate Cell Types}

\subsubsection{Novel distribution of cell types across strains}

%\begin{sidewaysfigure}
%\centering
%\begin{tabular}{cc}
%  \includegraphics[width=0.5\textwidth]{chapter5/DM_180613_AP_RatBlood_WKY_1a.pdf} & \includegraphics[width=0.5\textwidth]{chapter5/DM_180613_AP_RatBlood_SHR_1a.pdf}  \\
%\end{tabular}
%\caption{Representative plot of WKY and SHR blood FACS gating Results}
%\label{fig:FACS_BloodWKY}
%\end{sidewaysfigure}

The initial sorting protocol revealed novel differences in Leukocyte abundance between the WKY and SHR bloods (Figure ~\ref{fig:FACS_LeukPop}; Table ~\ref{fig:FACS_LeukPop_stats}). 

Cells expressing CD3+ appeared subtly more abundant within the WKY blood than within the SHR blood, however this was by a small degree and did not reach significance  (Figure ~\ref{fig:FACS_LeukPop_stats}; WKY vs SHR, -0.21 Fold, $\textit{P-Value=0.254}$). 

The CD4+ marker was again subtly different within the WKY blood than in the SHR blood, but this appears to be due to random spread (Figure ~\ref{fig:FACS_LeukPop_stats}; WKY vs SHR, -0.14 Fold, $\textit{P-Value=0.451}$).

There was a significant downregulation of CD8+ cells in the SHR blood, with a fold change of -0.36 (Figure ~\ref{fig:FACS_LeukPop_stats}; WKY vs SHR, -0.32 Fold, $\textit{P-Value=0.039}$).

B-Cell populations did not appear to vary between the two strains, and remained relatively constant at around 16\% of the total leukocyte population (Figure ~\ref{fig:FACS_LeukPop_stats}; WKY vs SHR, +0.04 Fold, $\textit{P-Value=0.704}$). 

Interestingly, total monocytes showed a trend towards increased abundance within the SHR. Whilst this was not significant, the breakdown of all monocytes into CD43Hi/His48Lo and CD43Lo/His48Hi subpopulations revealed the CD43Hi/His48Lo monocytes to be significantly upregulated in the SHR (Figure ~\ref{fig:FACS_LeukPop_stats}; WKY vs SHR, Monocytes +0.61 Fold, $\textit{P-Value=0.053}$; CD43Hi/His48Lo +0.96 Fold, $\textit{P-Value=0.014}$; CD43Lo/His48Hi +0.37, $\textit{P-Value=0.287}$). 

The FACS gating strategy for Neutrophils showed very little variation between the two strains, with both having neutrophils accounting for approx. 18\% of the total leukocyte population (Figure ~\ref{fig:FACS_LeukPop_stats}; WKY vs SHR, -0.14 Fold, $\textit{P-Value=0.670}$). 

Finally, the strongest change in abundance was observed with the NK Cells as the WKY expressed negligible levels whereas NK Cells accounted for approx. 3\% of the total leukocyte population in the SHR (Figure ~\ref{fig:FACS_LeukPop_stats}; WKY vs SHR, $\textit{P-Value$<$0.001}$). \\

\begin{figure*}[!htbp]
\centering
\includegraphics[width=0.9\textwidth]{chapter5/Leukocytes_pop.pdf}
\caption[Differences in leukocyte populations across \acrshort{wky} and \acrshort{shr} strains]{Differences in leukocyte populations across \acrshort{wky} and \acrshort{shr} strains. Statistical analysis made use of a Student's T-Test (P-Value $<$0.05, *; $<$0.01, **; $<$0.001, ***;  $<$0.0001, ****).}
\label{fig:FACS_LeukPop}
\end{figure*}

\begin{sidewaystable}[!htbp]
\scriptsize
\centering
\begin{tabular}{lrrrrrrrrr}
               & \multicolumn{9}{c}{Percentage of Total Leukocytes}                                                           \\
                          & CD3+    & CD4+    & CD8+    & B Cells & Monocytes & CD43Hi/His48Lo & CD43Lo/His48Hi & Neutrophils & NK Cells \\
               \hline
\textbf{SHR}              & 25.50\% & 16.50\% & 8.28\%  & 16.70\% & 28.90\%   & 20.00\%        & 6.42\%         & 21.80\%     & 3.09\%   \\
                          & 36.80\% & 24.80\% & 10.70\% & 16.80\% & 21.40\%   & 15.50\%        & 4.21\%         & 19.60\%     & 2.58\%   \\
                          & 45.50\% & 30.10\% & 12.40\% & 19.10\% & 19.20\%   & 14.60\%        & 3.03\%         & 9.76\%      & 2.84\%   \\
Mean                      & 35.93\% & 23.80\% & 10.46\% & 17.53\% & 23.17\%   & 16.70\%        & 4.55\%         & 17.05\%     & 2.84\%   \\
\textbf{WKY}              & 51.80\% & 31.90\% & 18.80\% & 18.90\% & 11.60\%   & 6.54\%         & 3.11\%         & 14.60\%     & 0.00\%   \\
                          & 46.70\% & 28.80\% & 16.80\% & 18.00\% & 16.10\%   & 9.98\%         & 3.14\%         & 15.50\%     & 0.00\%   \\
                          & 38.00\% & 22.70\% & 13.50\% & 13.40\% & 15.30\%   & 9.07\%         & 3.69\%         & 29.40\%     & 2.66E-05\% \\
Mean                      & 45.50\% & 27.80\% & 16.37\% & 16.77\% & 14.33\%   & 8.53\%         & 3.31\%         & 19.83\%     & 8.87E-06\% \\
\textbf{P-Value (T-Test)} & 0.254   & 0.451   & 0.039   & 0.704   & 0.053     & 0.014          & 0.287          & 0.670       & $<$0.001   
\end{tabular}
\caption[Statistical Comparison of Leukocyte Populations in WKY and SHR blood]{Statistical Comparison of Leukocyte Populations in WKY and SHR blood. Shown here are the relative abundances of cell types, taken from the total live leukocyte population. Statistical analysis here utilises the unpaired Student's T-Test.}
\label{fig:FACS_LeukPop_stats}
\end{sidewaystable}


\subsubsection{Assessment of Ifit1 expression within the cell subtypes}

Initial FACS isolation of cell populations for later qPCR analysis proved unsuccessful, as RNA levels were barely detectable across the samples. Nucleic acid concentrations ranged from 0 - 9ng/$\mu$l, with low accompanying 260:280 values. This was therefore prohibitive for taking samples forward for qPCR analysis of Ifit1 expression. It therefore led to a change in approach towards positive selection of Ifit1 containing cells by the \acrshort{facs} equipment.

\subsection{FACS Analysis on Permeabilised Cells}

\acrshort{facs} analysis was conducted on permeabilised leukocyte cells, incubated with the biotinylated anti-Ifit1 antibody. When applying the gates as originally optimised in section \ref{ValidationBiotinylatedAnti-Ifit1}, very few cells showed a positive signal (Figure \ref{fig:Ifit1pos}). This lack of positive signal resulted in very few cells able to pass through the various other gates used for determining leukocyte population (Figure \ref{fig:FACS_perm}). \\

\begin{figure*}[!htbp]
\centering
\includegraphics[width=0.6\textwidth]{chapter5/Ifit1Pos.png}
\caption[\acrfull{facs} output of BV605 Positive cells (Stained with anti-Ifit1 antibodies)]{\acrfull{facs} output of BV605 Positive cells (Stained with anti-Ifit1 antibodies), illustrating how few yielded a positive signal.}
\label{fig:Ifit1pos}
\end{figure*}




\section{Discussion}

\subsection{Ifit1 Role in Hypertension}

A growing body of evidence is suggesting the importance the innate immune system plays in several inflammatory processes known to affect the cardiovascular system. Innate immunity refers to the body’s first line of non-specific defense mechanisms against pathogens. However, the innate immune system also plays an important part in responding to endogenous damage. Increasing research is highlighting Ifit proteins in the innate immune response, specifically the Ifit1 protein. In 2012, McDermott \textit{et al.} used several large-scale transcriptomic datasets from mouse and mouse macrophage studies to identify a series of targets important in controlling innate immune responses \cite{McDermott2012}. By comparing these datasets, the team were able to infer networks from statistical associations between transcript expression profiles. From these networks, genes with a high-betweenness centrality were deemed important "bottlenecks" of the innate immune response. Across all the datasets analysed, they found only six conserved bottleneck genes including; Ifi47, Axud1, Ppp1r15a, Tgtp, Ifit1, and Oasl2. They continued to validate Ifit1 through qPCR analysis of expression in model of LPS-mediated TLR4 activation. Using Ifit1 siRNA expression of the predicted first-order network genes was robustly reduced, suggesting Ifit1 exerts an important regulatory influence across several downstream immune genes in this model of infection. 

The method of inducing the innate immune response by way of LPS exposure is a common one. However studies on LPS and blood pressure appear to yield mixed results. Many studies into the acute effects of LPS observe a reduction in blood pressure as a result of the infection, accompanied by a steady increase in heart rate to account for the lower blood pressure \cite{Fan2019}. Another study into the time course effects of hemodynamic responses to LPS noted a rapid reduction of blood pressure after a low dose of LPS (0.06mg/kg) was injected. Interestingly, the same team saw a significant increase in blood pressure at doses between 20-40mg/kg \cite{Brognara2019}. Chronic studies on LPS are less revealing, as animals rapidly desensitise to the exposure of LPS and as a result blood pressure remains unchanged for the duration of the study \cite{Lew2013}. 

It would certainly be of interest to manipulate this system in future, with LPS exposure at a higher level to see the effect it has on blood pressure and whether it is accompanied with a concomitant rise in Ifit1 expression occurs within the blood.

\subsection{Ifit1 in the \acrfull{pvn}}

Here a significant upregulation of Ifit1 expression was observed within the \acrshort{pvn}. The hypothalamic \acrshort{pvn} has been heavily implicated in the relationship between innate immunity and hypertension. Studies have shown increased levels of pro-inflammatory cytokines (PICs) to play an important factor in the development of hypertension. Shi \textit{et al.} (2010) show the involvement of microglial cytokines in neurogenic hypertension by administering either an anti-inflammatory antibiotic, or an adeno-associated virus for the overexpression of the anti-inflammatory IL-10 alongside angiotensin II in the \acrshort{pvn}. The team measured a series of physiological metrics associated with a hypertensive disease state including; mean arterial pressure, cardiac hypertrophy, and plasma norepinephrine after chronic angiotensin II administration. They found a significant reduction in all of these parameters after both of the treatments as compared with \acrshort{pbs} or AAV-GFP controls. The team further quantify a significant increase in activated microglia, as defined by increase OX42 antibody binding, the neuroglial cells tasked with innate immune responses to pathogens. Taken together these results highlight an important role for microglia and pro-inflammatory cytokines in AngII induced hypertension within the PVN (\cite{Shi2010}). For this reason, coupled with its other important regulatory roles outlined above, the \acrshort{pvn} was a key region for investigating Ifit1 involvement in hypertension. 
Indeed a case could be made for Ifit1 being a consequence of \acrfull{tlr4} signal transduction within the \acrshort{pvn}. One of the key aspects of innate immunity is the recognition of \acrfull{pamp}s in order to exact an immediate and coordinated attack on possible pathogens. Cells do this via \acrfull{prrs}, such as that of the \acrfull{tlrs}. The toll surface protein was initially discovered in Drosophila and characterised for its developmental role in determining the dorsal-ventral axis in embryos \cite{GAY1991}. Toll homologues in humans were described in the late 1990s, and became later renamed as \acrfull{tlrs}. At the time, there was no knowledge of the function they play in human biology. However, today they are known for their major role of the innate immune system, an attribute which can be exemplified by how evolutionarily conserved they are between species \cite{ONeill2013}. Most vertebrates have one ortholog for each of the \acrshort{tlrs} family, and sequencing experiments have revealed them in genomes as diverse as; chickens (\textit{Gallus gallus}), Japanese pufferfish (\textit{Takifugu rubripes}), and the opossum (\textit{Monodelphis domesticus}) \cite{Roach2005}. In total, 13 isoforms of TLRs have been identified in mammalian cells, with each recognising a specific \acrshort{pamp} \cite{Dange2015}. 

Upon activation \acrshort{tlrs} set off a diverse range biomolecular cascades. There are two major downstream branches of TLR activation that are characterised by the receptor adapter proteins they recruit; Myeloid Differentiation Primary Response Protein 88 (MyD88), and TIR-domain-containing Adapter Protein Inducing Interferon (TRIF). The latter leads to IRF3 activation, inducing the transcription of a broad range of ISGs; including Ifit family proteins \cite{SCHWABE20061886,doi:10.1002/hep.22306}. It is therefore plausible that Ifit1 upregulation could be consequence of \acrshort{tlr4} activation, and therefore an indirect consequence of a well documented mechanism of elevated blood pressure. 

Several studies have noted the pharmacological blockade of \acrshort{tlr4}, specifically within the \acrshort{pvn}, to attenuate blood pressure in a genetic model of hypertension. A study in 2015 used a bilateral inhibition of TLR4 within the PVN of the SHR and measured the blood pressure of the animals \cite{Dange2015}. Their study firstly revealed SHR rats to have a higher level of TLR4 in their PVN compared with WKY rats. Furthermore blockade of the TLR4 using a viral inhibitory peptide (VIPER) significantly lowered the blood pressure in the SHR animals and attenuated the cardiac hypertrophy often observed within SHRs. They found TLR4 inhibition to attenuate PICs (e.g. TNF-$\alpha$), induced Nitric Oxide Synthase (iNOS) and the activity of the NF-$\kappa\beta$ as well as an increase to levels of anti-inflammatory IL-10 within the PVN of hypertensive rats. Along with these observations, the team found a significant reduction to circulating plasma Noradrenaline in the SHRs, a circulating hormone responsible for the constriction of blood vessels to increase blood pressure \cite{Chistiakov2015}. As NF-$\kappa\beta$ was seen to be significantly reduced, the team suggest this attenuation to be attributable to a reduced inflammation, in turn leading to disruption of a detrimental positive feedback cycle involved in the progression of hypertension. Indeed recent studies have shown AngII induced hypertension to be blocked by NF-$\kappa\beta$ blockade in the PVN \cite{Cardinale2012}. The team continue to postulate the method by which TLR4 blockade to attenuate blood pressure as being via ultimately inhibiting sympathetic outflow. Noradrenaline is an indirect indicator of sympathetic activity, and was significantly reduced in the SHRs treated with the TLR4 blockade.  Additionally, TNF-$\alpha$ and NF-$\kappa\beta$ have both been implicated in cardiac remodeling in hypertension and end organ damage respectively \cite{Fu2004,Gupta2006}. Strengthening this hypothesis when taken together with the team's observations on cardiac hypertrophy.

A study in 2018 built upon these findings using the TAK-242 selective blocker of TLR4 in the PVN of SD rats fed on a low (0.3\%) or high-salt (8\%) diet \cite{10.1093/ajh/hpy074}. The team also noted elevated TLR4 expression in the PVN of the hypertensive rats, however this time in an environmentally induced form of hypertension. This was alongside a corresponding increase in reactive oxygen species and PICs. The team then used a different experimental method to block TLR4 expression, and again noted a resulting reduction in blood pressure. Excessive reactive oxygen species have a well established role in increasing sympathetic nerve excitability, ultimately leading to an elevated blood pressure \cite{Su2016}. These findings shed additional light on the method by which TLR4 activity translates to elevated blood pressure. While these studies did not study the resulting levels of Ifit1 expression in the PVN, they do implicate the innate immune system that has been associated to "bottleneck" upon this protein. These studies highlight the need for further analysis of ifit1's central role in blood pressure regulation in the PVN and suggest TLR4 activity in the PVN as having a potential involvement in ifit1 elevated expression. 

\subsection{FACS Analysis to Isolate Cell Types}
Here, focus was given to resolving the leukocyte population responsible for the elevated abundance of Ifit1 \acrshort{mrna} in the \acrshort{shr} blood. Whole blood contains several types of blood cell that can be broadly grouped into; erythrocytes, leukocytes, and thrombocytes. Of these, only leukocytes contain a nucleus and are therefore the subject of interest for studying elevated transcription of a transcript. However in 2009 a revolutionary study, that making use of microarray analysis, revealed the presence of over 1000 transcripts in a population of erythrocytes \cite{Kabanova2009}. This observation has clear ramifications towards studying the transcriptome of whole blood. While it should be noted that this study has not since been repeated, focus needs to be given to anucleated cells such as the erythrocytes and their progenitors (i.e. reticulocytes) when trying to deduce the source of a transcript of interest. Probing this 2009 dataset did reveal many the Ifit family of genes were detected by the microarray analysis. It may therefore be of use here to include erythrocytes in downstream analyses as they could be the source of the robustly elevated Ifit1 transcript. 

%Why not include CD68? \\
%A decision was made not to include the macrophage marker protein; CD68. The reasoning behind this was the poor expression of the CD68 protein on the cell surface (Insert citation). While this is an issue that can be mitigated by permeabilization and fixing, these additional steps would interfere with the RNA extraction protocol intended for the finally separated blood factions (insert citation). For this reason, identification of 

%Why use the SHR for antibody optimisation? 
Despite the high availability of rat bloods from other’s experiments, the decision was made to prioritise optimisation of the above protocol in SHR blood alone. The primary reason for this was the well documented increase in total leukocytes in SHR blood when compared to the WKY blood. A study from 1991 found an increase in leukocytes of between 50-100\% in young, mature, and old hypertensive rats compared with controls \cite{Schmid-Schonbein1991a}. The study further showed a 300\% increase in activated granulocytes, detected by a nitroblue tetrazolium reduction, in the SHR. These effects appeared consistent in rats obtained from colonies in Massachusetts, Denmark and [West] Germany. Therefore mitigating the various differences related to colony and seasonal variations in leukocyte counts.  For this reason, it seemed pertinent to optimise antibody dilutions for use in the SHR, so as to prevent; lack of binding due to low concentration or signal saturation due to over binding with a high concentration. 

A series of interesting observations arose from the initial FACS analysis of SHR and WKY blood cells. The relative abundances of specific leukocytes were significantly different between the two strains, notably the CD43 High/His48 Low monocytes, and the natural killer cells. These experiments were conducted with only 3 replicates, however the variance seen was sufficiently low to produce significant differences between the two cohorts. The observation of a differential abundance of leukocytes between these strains is not a new one, as many studies have noted the elevated levels of total leukocytes in the \acrshort{shr} as compared with the \acrshort{wky}. 

\subsubsection{NK Cells}
Interestingly, the \acrshort{shr} blood saw a significant increase in NK cells as compared with the \acrshort{wky} blood, where none were detected. Natural killer (NK) cells are non-B, non-T lymphocytes with the ability for "natural" antigen-independent cytotoxic activity and as such play a central role in the innate immune system \cite{Vivier2016}. NK cells are responsible for responding to abnormal or stressed tissue, and can activate or suppress inflammatory responses via their interactions with T-Cells, Dendritic cells, and macrophages \cite{Vivier2008}.With regards to hypertension, a mutual activation has been demonstrated between NK cells and monocytes in an angiotensin II-induced model of hypertension \cite{Kossmann2013}. This connection between NK cells and hypertension was furthered through the observations on vascular remodeling by Taherzadeh \textit{et al}, who used a congenic strain bred from C57BL/6 and BALB/c mice to show those sharing the same NK gene complex had a similar blood pressure response to a chronic \acrshort{lname}-induced hypertension model. This study highlights the potential role for NK cells in the sensitivity towards \acrfull{nos} inhibited forms of hypertension.

It has been shown in mice that angiotensin II induced inflammation and vascular dysfunction are associated with an accumulation of natural killer (NK) T cells in the aortic wall which, upon activation, rapidly release IFN-$\gamma$ \cite{Kossmann2013}. This angiotensin II caused effect is drastically reduced in $Tbx21^{-/-}$  a transcription factor that drives IFN production. Due to this massive upregulation in NK cells, coupled with the regulation of Ifit1 by IFNs, it appears quite probable they could be the source of the upregulated Ifit1 in the SHR blood. Either directly as the source of Ifit1, or indirectly through the upregulation of IFNs. 


\subsubsection{Monocytes}
Monocytes also represent another rapid responder to areas of inflammatory response, where upon activation differentiate to macrophages \cite{Ginhoux2014}. Unlike NK cells, a great deal of work has explored the relationship between macrophages and hypertension. Macrophages are always present within the vessel walls and within the kidney in hypertensives \cite{Rodriguez-Iturbe2017}. 

Via the detection of CD43 and His48, the total monocyte population appeared higher in abundance in the \acrshort{shr}, however this trend did not reach statistical significance (P=0.053). Based on the validation carried out in Barnett-Vanes \textit{et al.} (2016), these two cell surface markers are sufficient to differentiate between two monocyte subtypes; classical, and non-classical \cite{Barnett-Vanes2016}. In rats, the CD43 hi and lo monocytes are considered to be analogous to the mouse Ly6C Lo (non-classical) and Hi (classical) monocytes respectively \cite{Strauss-Ayali2007,Sunderkotter2004}. Classical (CD43hi/His48lo) monocytes represent the most abundant subtype, and are thought to represent newly released cells from the bone marrow \cite{10.1093/cvr/cvy112}. 

The non-classical (CD43lo/His48hi) monocytes are lower in abundance at around 40\% of the total circulating monocytes, until they expand in inflammatory states \cite{10.1093/cvr/cvy112}. This subtype is implicated as playing a role in patrolling the early immune response, tissue repair, and neovascularisation, as well as an increased production of TNF$\alpha$ \cite{Belge2002}. They have been dubbed the "sentinels" and "caretakers" of vascular tissue, due to their ability to recognise and clear dying endothelial cells, maintaining vascular homeostasis \cite{Carlin2013,Quintar2017}. While this non-classical subtype seem to be primarily involved in a homeostatic caretaking role they have also been seen to contribute to several autoimmune and inflammatory diseases, such as; rheumatoid arthritis, multiple sclerosis, and aetherosclerosis \cite{Narasimhan2019}. 

It was this non-classical subtype that saw a significant increase in abundance within the SHR blood, an observation that has not previously been noted in this model of hypertension. Indeed, this specific subtype has been implicated in human hypertension in a 2018 study conducted by Loperena \textit{et al.}, who used a FACS gating strategy to quantify monocyte populations in hypertensive and normotensives. The team observed a progressive decline in classical monocyte populations, and a concomitant increase in the percentage of non-classical monocytes as levels of hypertension increased \cite{10.1093/cvr/cvy112}. \\

The resulting cell populations were sorted directly into trizol, in an attempt to stabilise the cells and contained RNA as quickly as possible. Following a well established RNA extraction protocol, as outlined previously, the RNA yield was prohibitively low and prevented downstream qPCR analysis of Ifit1 expression levels. Due to time constraints, the samples for this experiment were immediately frozen at -80$\degree$C in order to stabilise them. After conducting a literature review to troubleshoot the protocol, several groups had noted a significant drop in RNA recovery following the freezing of the sample-trizol mix with some reporting as much as a 50\% reduction in RNA yield \cite{CellAnalysisFacility}.

A further concern could be the presence of RNases in the vicinity of the FACS machinery. It was not feasible to thoroughly prepare the instruments to reduce RNase activity. However, some protocols suggest adding an RNase inhibitors to the FACS suspension in order to mitigate any RNA degradation \cite{Barrett2002}. This approach presents a simple modification to the protocol followed above, and may well yield higher quality RNA for downstream analysis. 

\subsection{FACS Analysis on Permeabilised Cells}

Due to the resulting \acrshort{rna} being of poor quality, qPCR could not shed light on the relative abundances of Ifit1 expression within the different cell populations. The experiment did however yield a novel insight into the relative abundances of leukocytes, as outlined above. In order to overcome the poor \acrshort{rna} yield, a change in approach was attempted. Rather than physically isolating the live cells before extracting \acrshort{rna}, cells were permeabilised in order to allow antibodies to access the intracellular Ifit1. This way, Ifit1 positive cells could be detected and cross-referenced with the signal profiles of cell surface markers used to discern between leukocyte types. Furthermore, the act of permeabilisation required the fixation of the cells; thereby killing them. This fixation approach did not present as many concerns towards cell/transcript stability that were present for the live-cell sorting approach.

This approach was unsuccessful in resolving Ifit1 containing cell populations, due to the low levels of Ifit1 positive cells being detected. There could be many reasons behind this, all centering on one of two possibilities; either the experimental approach failed to detect the expressed Ifit1 protein, or Ifit1 protein is not expressed at levels comparable to the transcript.

It must be noted, that here previous analysis has been conducted on transcript levels of Ifit1 within the blood and PVN. A clear and robust increase in expression is occurring between the strains being tested, however this expression may not result in different levels of functional protein. A good review on the matter was conducted by Vogel and Marcotte (2012), who showed a correlation between expression levels and resulting proteins to be as little as 40\% \cite{Vogel2012}. There exist many processes of translational control between transcript expression and function protein, all of which present a stage at which a disparity can arise. mRNA stability, Protein stability, and the rate of mRNA transcription \textit{versus} protein translation all factor into the correlation observed. 

Focusing biomarker discovery work on transcripts is a valid approach if they stably and reliably act as a metric for physiological changes, as observed here. However, no conclusions can be drawn on Ifit1's specific role within hypertension without the use of a reliable antibody that is robustly validated. Transcriptomic analysis therefore presents a useful way to identify potential candidates for down-stream work at the level of the protein. 


\subsection{In Silico Characterisation of Ifit1}
Since the robust increase in ifit1 mRNA was present in the juvenile SHR at only 4 weeks, and previous blood pressure measurements of the SHR suggest it to be hypertensive at this age, it remains unclear whether Ifit1 is causative, resultant or neither with regards to hypertension. 

The high sequencing depth, coupled with the relatively large number of replicates, enabled me to look towards the NGS reads to resolve and nucleotide specific changes between the WKY and SHR Ifit1 mRNA in the blood. I was therefore able to characterise the genetic differences that exist between the WKY and SHR ifit1 gene, and furthermore whether these differences are responsive for the altered expression levels. With this in mind, and using the BAMtools suite of bioinformatic tools, I was able to merge all of the previously aligned BAM files from the juvenile WKY and SHR blood RNAseq experiment (n=6). This resulted in two BAM files containing all of the reads, aligned to the RN6 reference sequence, across all WKY samples and all SHR samples for a straightforward downstream analysis. The WKY and SHR compiled BAM files where sorted and indexed using the bamtools index command. Both the .bam aligned reads and the .bam.bai indexes could be read into the Integrative Genome Viewer (Version 2.4.4, Broad Institute California) \cite{Thorvaldsdottir2013, Glenn2011}. As these were both aligned and indexed against the same reference sequence (RN6), a simple search for “ifit1” localises the viewing panel to the locus of ifit1 in the rat genome. Once all reads are visualised against the ifit1 locus, a coverage histogram could be generated. Upon first assessment, and as expected with the predicted upregulation observed in chapter 1, there is a consistently higher number of RNAseq reads aligning to the gene from the SHR samples, than in the WKY samples. What is interesting, with regards to the WKY sample reads, is the relatively short span of ifit1 gene towards the 5’ that shows a high number of aligned reads  (reaching 1789 overlapping reads). 

The IGV software also allows for single nucleotide polymorphism calling (SNP), by comparing each read base against the reference genome. For the above coverage histogram, an allele frequency threshold can be arbitrarily set to reveal the site having that proportion of alleles called differently to the reference genome. Any loci whose proportion of reads are above this threshold will highlight as red on the coverage histogram (Table ~\ref{fig:SNPS}). The default threshold setting is left at 0.2, meaning over 20\% of reads on that particular locus have to differ from the reference base for the locus to highlight red. To increase stringency, and assess how prevalent any discovered SNPs are within each strain, I iteratively increased this threshold to inspect the resulting coverage profiles. At a threshold of 1.0, meaning an differential base calling event in every single read within the SHR population, 3 loci were highlighted as being different. \\

\begin{table}[!htbp]
\centering
\scriptsize
\begin{tabular}{lrccr}
Locus            & Percentage of reads & SNP   & Amino Acid Change & Intron/Exon \\
\hline
Chr1:252,944,180 & 100\% &  C $\rightarrow$ T & S $\rightarrow$ L & Exon \\
Chr1:252,944,345 & 87\%  &  C $\rightarrow$ T & S $\rightarrow$ L & Exon \\
Chr1:252,944,471 & 99\%  &  A $\rightarrow$ G & N $\rightarrow$ S & Exon \\
Chr1:252,944,488 & 100\% &  A $\rightarrow$ G & T $\rightarrow$ A & Exon \\
Chr1:252,945,968 & 100\% &  G $\rightarrow$ T & R $\rightarrow$ I & Exon \\
\end{tabular}
\caption{Location of single nucleotide polymorphisms in SHR Ifit1 transcripts, as compared against the RN6 reference genome}
\label{fig:SNPS}
\end{table}

However, it must be noted that as there were no reads from the WKY samples covering the above loci, it is not possible to assess whether these SNPs were truly part of a different genetic profile in the SHR. They may well be a part of a genetic profile shared between the strains, yet different from the RN6 reference genome the bases are being compared against. Further assessment of these potentially different SNPs could be achieved by PCR amplification of the ifit1 region, before Sanger sequencing to reveal any differences in the ifit1 gene between the WKY and SHR strains.  

When this threshold was dropped to 0.87 (87\% of reads), two more loci were highlighted. What is interesting is that one of these aligns with the only section of the ifit1 gene that saw a large number of reads within the WKY samples. At Chr1:252,944,345, an exchange of Cytosine $\rightarrow$ Thymine was observed in the SHR across those 87\% of reads covering that locus. For the same site for the WKY reads, no such SNP has taken place. This is therefore the only site where evidence exists to suggest a SNP existing between the WKY and SHR at this site.
  
At this point it should be noted that the sequence-specific error profile of RNAseq data is in no way 100\% accurate. This is a major consideration when attempting \textit{de novo} assembly or indeed the SNP calling attempted here. The potential issues with RNAseq inaccuracy are discussed earlier in greater length, however SNP calling is most reliant on preventing substitution-type miscalls of bases. A comparison study in 2011 looked into the various different sequencing platforms and estimated miscall rates to be $\geq$0.1\% of bases. This translates to a conservative 1 miscalled base in every 1000 bases sequenced \cite{Glenn2011}. While this number can be greater depending on platform used to sequence, our laboratory uses a Phred cutoff score of $>$Q30, translating to ~0.1\% miscalls per base. This threshold ensures the samples included in our analyses are at what is considered to be the current benchmark of accuracy given the library preparation chemistry at the time of sequencing. Illumina purports sequencing phred scores can often exceed this score, however it is seldom reported in the literature. Another consideration with base miscalls and phred score, is that this is calculated as an average of the entire read length. This becomes a problem because the miscall error rate tends to increase along the read length, as outlined in section ~\ref{Trimming} (Figure~\ref{fig:phasing_illumina}) \cite{Shendure2008}. For this reason, a read with a Phred Q30 score could on the surface appear sufficient to pass the stringent exclusion criteria for a given study. However, if this is an average of a read of Q40 between bases 1-100 and Q10 for the bases 101-150, the latter section of the read has a significantly large error rate of 1 miscall per 10 bases. This should therefore be a consideration when evaluating RNAseq data, and a level of stringency should exist that is sufficient to mitigate these error rates. Since these SNPs appear in $>$80\% of reads, coupled with a high read depth, for each loci it seems highly unlikely that these could be miscalls.

%\subsection{Concluding remarks}
%It is clear from these experiments, that Ifit1 presents an interesting avenue for future hypertensive research. 
%The continuation of this research, in characterising ifit1 further, is certainly sufficient to justify an entirely new body of work. 







